src/GafferOSL/OSLShader.cpp

```diff
@@ -468,19 +469,8 @@ Plug *loadCompoundNumericParameter( const OSLQuery::Parameter *parameter, const
 	}
 
 	typename PlugType::Ptr plug = new PlugType( name, parent->direction(), defaultValue, minValue, maxValue, Plug::Default | Plug::Dynamic, interpretation );
+	PlugAlgo::replacePlug( parent, plug );
```

Comment:
Let's do this in a separate commit, and do it consistently in all the `load*Parameter` functions, so we can actually remove the `transformConnectionOrValue()` function entirely.